.travis.yml

```diff
@@ -17,7 +17,7 @@ matrix:
       language: node_js 
       node_js: "6"
       jdk: oraclejdk8
-      script: cd demo && npm run build.plugin && npm i && npm run build-ios-bundle
+      script: cd demo && npm run build.plugin && npm i && tns build ios --bundle

```

Comment:
It would be nice if we add --env.uglify preparing the plugin to become verified one